src/provider.js

```diff
@@ -29,7 +29,8 @@ Provider.childContextTypes = {
 
 Provider.propTypes = {
   inspect: PropTypes.func,
-  state: PropTypes.object
+  state: PropTypes.object.isRequired,
+  children: PropTypes.node
```

Comment:
children should not be in prop types as we use a `render` prop instead.
I like it better because it's more explicit than `children`. It tells exactly what it does

Reply:
Ouch I am sorry I thought it was the `State` component.

Thanks for adding the children here, do you mind adding `isRequired` for it as well